airflow/include/tasks/extract/utils/weaviate/ask_astro_weaviate_hook.py

```diff
@@ -220,84 +222,91 @@ def batch_ingest(
         :param uuid_column: Name of the column containing the UUID.
         :param vector_column: Name of the column containing the vector data.
         :param batch_params: Parameters for batch configuration.
-        :param existing: Strategy to handle existing data ('skip', 'replace', 'upsert').
-        :param verbose: Whether to print verbose output.
+        :param existing: Strategy to handle existing data ('skip', 'replace', 'upsert' or 'error').
+        :param verbose: Whether to log verbose output.
+        :param tenant: The tenant to which the object will be added.
```

Comment:
I would suggest we keep the docstring order consistent with the parameters

Reply:
Done